lib/semantics/mod-file.cc

```diff
@@ -280,6 +289,7 @@ void ModFileWriter::PutUseExtraAttr(
 }
 
 // Collect the symbols of this scope sorted by their original order, not name.
+// Namelists are an exception: they are sorted to the end.
```

Comment:
What does "sorted to the end" mean?  Something about the fact that a later statement can add group objects to a group from a previous statement?  (8.9p3)

Reply:
It means the symbols for namelists appear at the end of the sorted vector of symbols, not in order of first occurrence of the name as with other symbols. This is so that the declarations of the objects in the namelist are sure to occur before the namelist statement.

I'll try to make the comment clearer.